components/o-message/src/tsx/message.tsx

```diff
@@ -0,0 +1,84 @@
+import { Fragment } from 'react';
+
+type MessageContent = {
+	detail: string;
+	hightLight?: string;
+	additionalInfo?: string;
+}
+
+type MessageAction = {
+	copy: string;
+	url: string;
+	openInNewWindow?: boolean
+}
+
+type MessageProps = {
+	type: 'action' | 'alert' | 'notice';
+	state: 'success' | 'neutral' | 'error' | 'feedback' | 'warning' | 'warning-light' | 'inform' | 'inform-inverse'
+	content: MessageContent;
+	primaryAction?: MessageAction;
+	secondaryAction?: MessageAction;
+	hasInnerLayout?: boolean;
+	showCloseButton?: boolean;
+	autoOpen?: boolean;
+	parentElement?: string;
+}
+
+type TextContentPropType = {
+	text: string;
+}
+
+const InnerLayout = ({ condition, wrapper, children }) => {
+	return condition ? wrapper(children) : children;
+}
+
+const AdditionalInfo = ({text}: TextContentPropType) => {
+	return <p className="o-message__content-additional">{text}</p>
+}
+
+const Highlight = ({text}: TextContentPropType) => {
+	return <span className="o-message__content-highlight"> {text} </span>
+}
+
+
+export function Message({
+	type = 'alert',
+	state = 'success',
+	content = undefined,
+	primaryAction = undefined,
+	secondaryAction = undefined,
+	hasInnerLayout=false,
```

Comment:
```suggestion
	inner=false,
```

let's make this the simpler `inner` to match the modifier class name